server/routes/nl/api.py

```diff
@@ -0,0 +1,569 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
Any changes being made to this file?  Or is it just a rename?

Reply:
just moving code from /nl/html.py